### ISSUE TITLE
Add own WADS, IWADs to prBoom

### ISSUE THREAD

**Issue by microton79** [webrcade/webrcade#21]
Hi!

Will it be possible to load other games/mods into prboom using a weblink like with the other applications within webrcade?

**Comment by raz0red** [comment]
Yes, it will be possible. There will ultimately be an option to point at your own wads, etc. similar to the other applications based on emulators.

**Comment by XargonWan** [comment]
In our retro firmware we use the .doom files to teach the engine how to behave: maybe this can be an idea of implementation. More details [here](https://351elec.de/System-Doom).
